R/pairwise.R

```diff
@@ -50,7 +50,10 @@ do_cosine_sim.kv <- function(df, subject, key, value = NULL, distinct=FALSE, dia
     df
   }
 
-  (df %>% dplyr::do_(.dots=setNames(list(~calc_doc_sim_each(.)), cnames[[1]])) %>%  tidyr::unnest_(cnames[[1]]))
+  tmp_col <- avoid_conflict(grouped_column, "tmp")
```

Comment:
It's good that this is done in simpler way, but still these line deserves a comment on what they are doing.